src/main/scala/io/digitallibrary/bookapi/service/ReadService.scala

```diff
@@ -92,6 +92,13 @@ trait ReadService {
       } yield converterService.toApiBook(translation, unFlaggedTranslationsRepository.languagesFor(bookId), book)
     }
 
+    def withIdAndLanguageListingAllBooksIfAdmin(bookId: Long, language: LanguageTag): Option[api.Book] = {
+      for {
+        translation <- getTranslationRepository.forBookIdAndLanguage(bookId, language)
```

Comment:
`getTranslationRepository` chooses a repository instance with/without privileges based on the logged in user's claims.